tests/testthat/test-github.R

```diff
@@ -0,0 +1,42 @@
+test_that("use_github_links populates empty URL field", {
+  local_interactive(FALSE)
+  create_local_package()
+  use_git()
+  skip_if_no_git_user()
```

Comment:
I would place the skip at the very top. I think that's a general principle, i.e. if you're going to skip, get out as early as possible.
And, in this specific case, definitely before `use_git()`.

Reply:
🤦